src/models/constants.ts

```diff
@@ -16,3 +16,5 @@ export const SERVER_ADDRESS = "https://dev-yu3covlh6a-uc.a.run.app";
 
 export const EMPTY_PROFILE_PIC =
   "https://t4.ftcdn.net/jpg/02/15/84/43/240_F_215844325_ttX9YiIIyeaR7Ne6EaLLjMAmy4GvPC69.jpg";
+
+export const NOTIFICATION_LENGTH = 3000;
```

Comment:
Just curious what this is for. The total number of notifications to load? The max length of a notification?

Reply:
Length to display the notification in ms